db/mgo/bson/extensions.go

```diff
@@ -9,10 +11,10 @@ func (d D) UnmarshalBSON(in []byte) error    { return Unmarshal(in, &d) }
 func (r RawD) MarshalBSON() ([]byte, error)  { return Marshal(r) }
 func (r RawD) UnmarshalBSON(in []byte) error { return Unmarshal(in, &r) }
 
-func (o *ObjectId) UnmarshalBSONValue(t bsontype.Type, in []byte) error {
+func (o *ObjectId) UnmarshalBSONValue(_ byte, in []byte) error {
 	*o = ObjectId(in)
 	return nil
 }
-func (o ObjectId) MarshalBSONValue() (bsontype.Type, []byte, error) {
-	return bsontype.ObjectID, []byte(o), nil
+func (o ObjectId) MarshalBSONValue() (typ byte, data []byte, err error) {
+	return byte(bson.TypeObjectID), []byte(o), nil
 }
```

Comment:
These interfaces changed in signature: https://github.com/mongodb/mongo-go-driver/blob/master/docs/migration-2.0.md#valuemarshaler